ethpm_cli/manifest.py

```diff
@@ -68,7 +74,7 @@ def generate_custom_manifest(project_dir: Path) -> None:
 
 def gen_validate_manifest() -> Optional[Callable[..., Manifest]]:
     flag = parse_bool_flag(
-        "Would you like to validate your manifest against the json schema? (recommended) (y/n) "
+        "Would you like to validate your manifest against the json schema? (recommended)"
```

Comment:
Maybe extend `parse_bool_flag` to support a `default` so that the user can just pres `Enter` without actually typing in a `y` or `n`.  My intuition says this would reduce mistakes where a user is repeatedly typing in `y` to get through the prompts and says `y` to something they don't intend to.

Reply:
Yeah, I could see that being helpful but maybe down the road after the wizard is a little more fleshed out. It's just not clear to me whether or not it should always default to `y`? Like for adding links to the manifest, I'm not sure we should default to making the user do that and every other possible field? Or should the default change depending on the specific question?

Reply:
The default would change depending on the specific question.